app/scripts/models/auth_brokers/base.js

```diff
@@ -175,6 +186,15 @@ define([
 
     _isForceAuthUrl: function () {
       return this.window.location.pathname === '/force_auth';
+    },
+
+    _isSignInUrl: function () {
+      var pathname = this.window.location.pathname;
+      return pathname === '/signin' || pathname === '/oauth/signin';
```

Comment:
> We could change the GET /authorization endpoint to handle this case and redirect us to the right endpoint to begin with (e.g., like it's doing with action=signin and action=signup).

The original goal of this PR was to have it done by Monday for inclusion in train 28. I used a content-server only solution to eliminate external deps to meet that goal. Since this is now aimed at the next train, making modifications to the oauth server makes more sense. That would make my reasoning in https://github.com/mozilla/fxa-content-server/pull/2019#discussion_r22647111 moot because the oauth server would handle the setup work.